static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js

```diff
@@ -5,11 +5,29 @@ import { Advert } from 'commercial/modules/dfp/Advert';
 import { getAdvertById } from 'commercial/modules/dfp/get-advert-by-id';
 import { enableLazyLoad } from 'commercial/modules/dfp/lazy-load';
 
-const viewabilityThresholdMs = 30000;
+const getViewabilityThreshold = (advert: Advert): ?number => {
+    const refreshThresholdMsDefault = 30000;
+    const refreshThresholdMsVideo = 90000;
+
+    const fluidSize = '0,0';
+
+    const sizeString = advert.size && advert.size.toString();
+    const videoSizes = ['620,1', '620,350'];
+
+    if (sizeString === fluidSize) {
+        return;
```

Comment:
What are you returning in this case? Does number default to zero?

Reply:
Returning nothing in this case - the return type of `?number` indicates that it is a potentially undefined.

Reply:
Ah sorry - missed the ?